admin_guide/high_availability.adoc

```diff
@@ -580,6 +588,51 @@ the list of VIPs.  The vrrp-ids may have values in the range 1..255.
 
 When there are multiple ipfailover deployment configuration care must be taken to specify `--vrrp-id-offset` so that there is room to increase the number of VIPS in the deployment configuration and none of the vrrp-id ranges overlap.
 
+[[admin-guide-high-availability-configuring-more-than-254]]
+=== Configuring IP failover for more than 254 addresses
+
+By default, IP failover has a limitation of managing only 254 VIP addresses. You
+can group sets of VIPs for VRRP instances using the `virtual-ip-groups` option
+to define the number of VIP groups available for each VRRP instance when
+cxref:../admin_guide/high_availability.adoc#configuring-ip-failover[configuring
+IP failover]. 
+
+This creates a wider range of allocation of VIPs per VRRP in the case of VRRP
+failover events, and is useful when all hosts in the cluster have access to a
+service locally. For example, when a service is being exposed with an external
+IP. 
+
+[NOTE]
+====
+As a rule for failover, services, such as the router, should not be limited to
+one specific host. Instead, services should be replicated to each host so that
+in the case of IP failover, the services do not have to be recreated on the new
+host.
+
+If you are using {product-title} health checks, the nature of IP failover and
+groups means that all instances in the group will not be checked. For that
+reason, the Kubernetes health checks will be more effective.
+====
+
+----
+# oc adm ipfailover <ipfailover_name> --create \
+    ...
+    --virtual-ip-groups=<number_of_ipfailover_groups>
+----
+
+For example, if the above is used in a environment with seven VIPs, it creates
+three groups, assigning three VIPs to the first group, and two VIPs to the two
```

Comment:
Don't you want to hard-code '--virtual-ip-groups=3' on line 620 ? Otherwise this example does not explain why it creates three groups.